packages/web/src/components/buy-sell-modal/BuySellModal.tsx

```diff
@@ -22,36 +22,42 @@ export const BuySellModal = () => {
 
   // State to control the modal title based on the flow screen
   const [modalScreen, setModalScreen] = useState<'input' | 'confirm'>('input')
+  const [isFlowLoading, setIsFlowLoading] = useState(false)
+
+  const title = useMemo(() => {
+    if (isFlowLoading) return ''
```

Comment:
is this gonna look weird? will it flicker when we're moving between states?

Reply:
nah not seeing any flicker or anything but will keep an eye out